### PR TITLE
Update sensor.bbox configuration varibales

### PR DESCRIPTION
**Pull request by Mryck**
**Description:**

Update bbox sensor #6385

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#<home-assistant PR number goes here>

## Checklist:

- [x] Branch: `next` is for changes and new documentation that will go public with the next [home-assistant](https://github.com/home-assistant/home-assistant) release. Fixes, changes and adjustments for the current release should be created against `current`.
- [x] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

### CODE REVIEW

File: source/_components/sensor.bbox.markdown [hunk 1]
```diff
@@ -34,11 +34,18 @@ sensor:
...
 
-Configuration variables:
-
-- **monitored_variables** array (*Required*): Sensors to display in the frontend.
-  - **down_max_bandwidth**: Maximum bandwidth available for download.
-  - **up_max_bandwidth**: Maximum bandwidth available for upload.
-  - **current_down_bandwidth**: Instant measure of the current used bandwidth for download.
-  - **current_up_bandwidth**: Instant measure of the current used bandwidth for upload.
-
+{% configuration %}
```
Comment (klaasnicolaas) [inline source/_components/sensor.bbox.markdown:37]: Pherhaps you could also add the variable `name`, [see more](https://github.com/home-assistant/home-assistant/blob/dev/homeassistant/components/sensor/bbox.py#L47)

Comment (Mryck) [reply source/_components/sensor.bbox.markdown:37]: Well spotted @klaasnicolaas thanks

**Review by klaasnicolaas — APPROVED** [review]

File: source/_components/sensor.bbox.markdown (hunk 1 above)
Comment (klaasnicolaas) [reply source/_components/sensor.bbox.markdown:37]: 👍 No problem 😄

**Review by frenck — APPROVED** [review]

**Comment by frenck** [comment]
Thanks for the PR @Mryck!

And @klaasnicolaas, thanks for helping us with the reviews, that is appreciated.

Happy Hacktoberfest 🎉 

../Frenck

**Pull request merged**